edm4eic.yaml

```diff
@@ -140,6 +140,13 @@ components:
       - edm4eic::Cov2f    directionError  // Error on the polar and azimuthal angles
       - float             pathlength      // Pathlength from the origin to this point
       - float             pathlengthError // Error on the pathlenght
+
+  ## PID hypothesis from Cherenkov detectors
+  edm4eic::CherenkovPdgHypothesis:
+    Members:
+      - int32_t           pdg             // PDG code
+      - float             npe             // Overall photoelectron count
+      - float             weight          // Likelihood estimate
```

Comment:
@alexander-kiselev or @chchatte92: does IRT's `MassHypothesis::GetWeight` return the likelihood or log(likelihood)? What do we want to store in the data model, likelihood or log(likelihood)?

Reply:
I changed the description to `Weight of this hypothesis, such as likelihood, moment, etc.` to make it more generic. Currently our weights from IRT v1.0.6 are more like moments, but I believe this will change again with the new IRT version (on branch `pfrich` from @alexander-kiselev's pfRICH work, to be eventually released as `v1.1.0+`, not targeting this 'end-of-march' campaign).